cli/internal/terraform/loader.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package terraform
+
+import (
+	"embed"
+	"errors"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
+	"github.com/edgelesssys/constellation/v2/internal/file"
+	"github.com/spf13/afero"
+)
+
+//go:embed all:terraform/*
```

Comment:
```suggestion
//go:embed terraform/*
```

All `all:` does is including hidden files (files starting with `.` or `_`). Shouldn't matter for release builds built in a clean Linux environment but can either include trash (e.g. `.DS_Store` on macOS) or even worse: a `.terraform` directory from running Terraform directly in the directory to be embedded that overshadows the real filesystem. 

That can cause some funny non-sense and break stuff in weird ways. Ask me how I know...